zilsd.adoc

```diff
@@ -98,19 +100,31 @@ It is implementation defined whether interrupts can also be taken during the seq
 
 === Software view of the load/store pair sequence
 
-From a software perspective the load/store pair instructions appears as:
+From a software perspective load/store pair instructions appear as:
 
 * load instructions:
-** A sequence of one or more loads reading the bytes of the double word without updating rd or rd+1
-*** The bytes may be loaded in any order.
-*** The bytes may be grouped into larger accesses.
-*** Any of the bytes may be loaded multiple times.
+** A sequence of one or more loads reading the bytes of the doubleword without updating rd or rd+1
+*** If the effective address is 4B aligned:
+**** The bytes are grouped into word accesses.
+**** The words may be loaded in any order.
+**** The words may be grouped into doublewords accesses.
+**** Any of the words may be loaded multiple times.
+*** Else:
+**** The bytes may be loaded in any order.
+**** The bytes may be grouped into larger accesses.
+**** Any of the bytes may be loaded multiple times.
 ** An atomic write of the load result into rd and rd+1
 * store instructions:
-** A sequence of one or more stores writing the bytes of the double word
-*** The bytes may be stored in any order.
-*** The bytes may be grouped into larger accesses.
-*** Any of the bytes may be stored multiple times.
+** A sequence of one or more stores writing the bytes of the doubleword
+*** If the effective address is 4B aligned:
+**** The bytes are grouped into word accesses.
+**** The words may be stored in any order.
+**** The words may be grouped into larger accesses.
```

Comment:
Lets tighten this a bit. The words may be grouped into a doubleword. Just to avoid someone creating a 5B + 1B grouping.

Reply:
that was unintended copy paste error. fixed

Reply:
should also be fixed for the stores.

Reply:
sorry, i did fix it but failed to push. i think it looks fine now